src/parser/parser.yy

```diff
@@ -827,6 +827,9 @@ function_call_expression
     | KW_DATETIME L_PAREN opt_argument_list R_PAREN {
         $$ = new FunctionCallExpression(new std::string("datetime"), $3);
     }
+    | KW_TAGS L_PAREN opt_argument_list R_PAREN {
+        $$ = new FunctionCallExpression(new std::string("tags"), $3);
+    }
```

Comment:
Why we had this function a standalone rule?

Reply:
support to use `tags(v)` in expression directly.